Gemfile.lock

```diff
@@ -112,4 +112,4 @@ DEPENDENCIES
   wdm (>= 0.1.0)
 
 BUNDLED WITH
-   2.4.21
+   2.4.21
```

Comment:
do not commit this❌

Reply:
This Commit was done to restore back to the original installation settings. System made changes to the gemfile while doing the setup.

Reply:
@shubhusion in other words, there is no _actual_ change here. Ok, that's good. 

Ideally, this change is removed from this PR as it's unnecessary churn.

Reply:
Yeah